Makefile

```diff
@@ -77,7 +77,8 @@ CONFIG_RTW_SDIO_PM_KEEP_POWER = y
 ###################### MP HW TX MODE FOR VHT #######################
 CONFIG_MP_VHT_HW_TX_MODE = n
 ###################### Platform Related #######################
-CONFIG_PLATFORM_I386_PC = y
+CONFIG_PLATFORM_I386_PC = n
+CONFIG_PLATFORM_ARM_RPI = y
```

Comment:
The change is good, thanks. However changing the PLATFORM setting will break the build for everyone else. Can the change at lines 80/81 be modified to keep "CONFIG_PLATFORM_I386_PC = y" and to set "CONFIG_PLATFORM_ARM_RPI = n"?

Reply:
Sure great suggestion!

Reply:
Hi, do you mind updating your merge request to that effect?